misc/package/Data Files/MWSE/core/lib/event.lua

```diff
@@ -76,45 +78,47 @@ local function remapFilter(options, showWarnings)
 	options.filter = filter.id:lower()
 end
 
+
+local doOnceCallbacks = {} ---@type table<fun(...):..., fun(...):...>
+
+
 function this.register(eventType, callback, options)
-	-- Validate event type.
-	if (type(eventType) ~= "string" or eventType == "") then
-		return error("event.register: Event type must be a valid string.")
-	end
```

Comment:
Why did you remove this check?

Reply:
I removed those checks because they're happening in `event.isRegistered` and the newer implementation calls `event.isRegistered` from inside `event.register`.